### PR TITLE
[feature](stats) Support sync analyze

### PR DESCRIPTION
**Pull request by Kikyou1997**
# Proposed changes

Gammer:

```
ANALYZE [SYNC] TABLE ....
```

Add this feature so that we could test and tune stats framework conveniently.

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 36.01 seconds
 stream load tsv:          422 seconds loaded 74807831229 Bytes, about 169 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          73 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230411132539_clickbench_pr_127617.html

File: regression-test/suites/statistics/analyze_test.groovy [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    );"""
+
+
+    sql """insert into test_table_alter_column_stats values(1, 2, 3);"""
+    sql """insert into test_table_alter_column_stats values(4, 5, 6);"""
+    sql """insert into test_table_alter_column_stats values(7, 1, 9);"""
+    sql """insert into test_table_alter_column_stats values(3, 8, 2);"""
+    sql """insert into test_table_alter_column_stats values(5, 2, 1);"""
+
+    sql """delete from __internal_schema.column_statistics where col_id in  ('col1', 'col2', 'col3')"""
```
Comment (englefly) [inline regression-test/suites/statistics/analyze_test.groovy:38]: this deletion may affect other test case, if that case depends on statistics and column name is in { col1, col2, col3}
Please rename column name to a more specific name, 'col1' is too general.

**Comment by englefly** [comment]
1. can we cancel this sync task by ctrl+C ?
2. after sync task launched, if I close the mysql client terminal, this task will be terminated or continue?

**Review by englefly — APPROVED** [review]
Do not forget to rename column name in the next pr.

**Pull request merged**